### PR TITLE
Updated models fields to not  use settings directly.

### PR DESCRIPTION
**Pull request by ipanova**
closes #7728
https://pulp.plan.io/issues/7728

### CODE REVIEW

File: CHANGES/7728.misc [hunk 1]
```diff
@@ -0,0 +1 @@
+Updated models fields to not  use settings directly.
```
Comment (mdellweg) [inline CHANGES/7728.misc:1]:
```suggestion
Updated models fields to not use settings directly.
```

**Review by mdellweg — APPROVED** [review]

File: pulp_container/app/migrations/0009_get_upload_path.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+        ('container', '0008_include_exclude_tags'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='upload',
+            name='file',
+            field=models.FileField(max_length=255, null=True, storage=django.core.files.storage.FileSystemStorage(location=pulp_container.app.models.Upload._get_upload_path), upload_to=pulp_container.app.models.generate_filename),
+        ),
+    ]
```
Comment (mdellweg) [inline pulp_container/app/migrations/0009_get_upload_path.py:20]: Fun fact: I think this migration adheres to the yet to define zero-downtime policy. :+1:

**Review by mdellweg — APPROVED** [review]
Apart from black. ;)

**Comment by ipanova** [comment]
> Apart from black. ;)

damn these double quotes!

**Pull request merged**